app/controllers/concerns/form_dradifier.rb

```diff
@@ -0,0 +1,39 @@
+module FormDradifier
+  extend ActiveSupport::Concern
+
+  def convert_form_content
+    return unless params[:item_form]
+
+    item = instance_variable_get("@#{controller_name.singularize}")
+    # Assign the issue for the Issues::MergeController
+    item = @issue if controller_name.singularize == 'merge'
+
+    content_attribute =
+      case item
+      when Card; :description
```

Comment:
One might argue this block knows to much about other classes. Where the ruby way would be to use duck-typing. I'm impartial on type-safety vs duck typing arguments so I'll let you decide.

Reply:
I'd like to go with adding an alias method for the models, but I remember @etdsoft not being a fan of this approach because ideally we should have a uniform naming for the model attributes.
I could also go with the approach of the controllers declaring this attribute but this ties in with our previous conversation that controllers are doing too much just to include this module.
So, I'm electing to keep this as is.